nix/pkgs.nix

```diff
@@ -0,0 +1,45 @@
+{ pkgs ? import <nixpkgs> {}
+, iohk-extras ? {}
+, iohk-module ? {}
+, haskell
+, ...
+}:
+let
+
+   # our packages
+  stack-pkgs = import ./.stack-pkgs.nix;
```

Comment:
I think this needs to be `./.stack.nix/default.nix`. Because the output filenames of nix-tools changed, that one is stale.